webui/src/pages/repositories/repository/tags.jsx

```diff
@@ -0,0 +1,236 @@
+                            {tag.commit_id.substr(0, 12)}
+                        </LinkButton>
+                        <ClipboardButton variant={buttonVariant} text={tag.id} tooltip="Copy ID to clipboard" />
+                        <ClipboardButton variant={buttonVariant} text={`lakefs://${repo.id}/${tag.id}`} tooltip="Copy URI to clipboard" icon={<LinkIcon />} />
+                        <ClipboardButton variant={buttonVariant} text={`s3://${repo.id}/${tag.id}`} tooltip="Copy S3 URI to clipboard" icon={<PackageIcon />} />
+                    </ButtonGroup>
+                </div>
+            </div>
+        </ListGroup.Item>
+    );
+};
+
+
+const CreateTagButton = ({ repo, variant = "success", onCreate = null, children }) => {
+    const [show, setShow] = useState(false);
+    const [disabled, setDisabled] = useState(false);
+    const [error, setError] = useState(null);
+    const textRef = useRef(null);
+    const defaultBranch = useMemo(
+        () => ({ id: repo.default_branch, type: "branch" }),
+        [repo.default_branch]);
+    const [selectedBranch, setSelectedBranch] = useState(defaultBranch);
+
+    const hide = () => {
+        if (disabled) return;
+        setShow(false);
+    };
+
+    const display = () => {
+        setShow(true);
+    };
+
+    const onSubmit = () => {
+        setDisabled(true);
+        const tagId = textRef.current.value;
+        const sourceRef = selectedBranch.id;
+        tags.create(repo.id, tagId, sourceRef)
+            .catch(err => {
+                setError(err);
+            })
+            .then((response) => {
+                setError(false);
+                setDisabled(false);
+                setShow(false);
+                if (onCreate !== null) onCreate(response);
+            });
+    };
+
+    return (
+        <>
+            <Modal show={show} onHide={hide}>
+                <Modal.Header closeButton>
+                    Create Tag
+                </Modal.Header>
+                <Modal.Body>
+
+                    <Form onSubmit={(e) => {
+                        onSubmit();
+                        e.preventDefault();
+                    }}>
+                        <Form.Group controlId="name" className="float-left w-25">
+                            <Form.Control type="text" placeholder="Tag Version" name="text" ref={textRef} />
+                        </Form.Group>
+                        <Form.Group controlId="source">
+                            <span className="ml-2 mr-2">@</span>
+                            <RefDropdown
+                                repo={repo}
+                                emptyText={'Select Branch'}
+                                selected={selectedBranch}
+                                selectRef={(refId) => {
+                                    setSelectedBranch(refId);
+                                }}
+                                withCommits={true}
+                                withWorkspace={false} />
```

Comment:
I can't filter branches using this dropdown, is it on purpose?

Reply:
Hey @itaiad200 not able to figure out what's causing this. I'm not able to use the filter option in the Create Branch modal in the branches page either

Reply:
True, I can't either. Unrelated to this PR.
Opened #2460 and resolving this comment.